### PR TITLE
fix the warns from cargo clippy

### PR DESCRIPTION
**Pull request by utam0k**
Signed-off-by: utam0k <k0ma@utam0k.jp>

### CODE REVIEW

**Comment by YJDoc2** [comment]
Hey, thanks a lot for opening this :hugs: . As this is not marked WIP, I'll  take a look at this and merge if no issue :)

**Review by saschagrunert — APPROVED** [review]

File: crates/libcontainer/src/rootfs/mount.rs [hunk 1]
```diff
@@ -378,7 +374,7 @@ impl Mount {
         if let Some(l) = label {
             if typ != Some("proc") && typ != Some("sysfs") {
                 match mount_option_config.data.is_empty() {
-                    true => d = format!("context=\"{}\"", l),
+                    true => d = format!("context=\"{l}\""),
                     false => d = format!("{},context=\"{}\"", mount_option_config.data, l),
```
Comment (YJDoc2) [inline crates/libcontainer/src/rootfs/mount.rs:378]: Hey, why is this part not formatted in the same way? we can format `l` in the same way here right?

**Review by YJDoc2 — APPROVED** [review]
lgtm :+1:
Just check the one comment I have left

File: crates/libcontainer/src/rootfs/mount.rs (hunk 1 above)
Comment (utam0k) [reply crates/libcontainer/src/rootfs/mount.rs:378]: I'm not sure because this changes created by cargo fmt and clippy

**Pull request merged**